### PR TITLE
feat: Add example for Gemini batch predictions using BigQuery input

### PR DESCRIPTION
**Pull request by gericdong**
# Description

Add example for Gemini batch predictions using BigQuery input.
We already have a example for batch prediction using Cloud Storage input.

### CODE REVIEW

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb
Comment (holtskinner) [inline gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:1]: Overall comment - Would it make sense to merge this with the existing intro to batch prediction notebook? https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/batch-prediction/intro_batch_prediction.ipynb

Just add another section that shows BQ input so they don't need to re-do all of the steps?

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+        "The input for batch requests specifies the items to send to your model for prediction.\n",
+        "\n",
+        "Batch requests for Gemini accept BigQuery storage sources and Cloud Storage sources. You can learn more about the batch input formats for BigQuery and Cloud Storage sources in the [Batch text generation](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/batch-prediction-gemini#prepare_your_inputs) page.\n",
+        "\n",
+        "This tutorial uses **BigQuery** as an example. To use a BigQuery table as the input, you must ensure the following:\n",
+        "\n",
+        "- The BigQuery dataset must be created in a specific region (e.g. `us-central1`). Multi-region location (e.g. `US`) is not supported.\n",
+        "- The input table must have a column named `request` in JSON or STRING type.\n",
+        "- The content in the `request` column must be valid JSON. This JSON data represents your input for the model.\n",
+        "- The content in the JSON instructions must match the structure of a [GenerateContentRequest](https://cloud.devsite.corp.google.com/vertex-ai/generative-ai/docs/model-reference/inference#request_body).\n",
```
Comment (holtskinner) [inline gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:317]:
```suggestion
        "- The content in the JSON instructions must match the structure of a [`GenerateContentRequest`](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference#request_body).\n",
```

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+        "- Only public YouTube and Cloud Storage bucket URIs in the `fileData` or `file_data` field are supported in batch prediction.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3d4b751610f0"
+      },
+      "source": [
+        "This is an example BigQuery table:"
```
Comment (holtskinner) [inline gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:328]: Is this table publicly accessible?

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+        "When a batch prediction task completes, the output is stored in the location that you specified in your request.\n",
+        "\n",
+        "- The location is in the form of a Cloud Storage or BigQuery URI prefix, for example:\n",
+        "`gs://path/to/output/data` or `bq://projectId.bqDatasetId`.\n",
+        "\n",
+        "- If not specified, `STAGING_BUCKET/gen-ai-batch-prediction` will be used for Cloud Storage source and `bq://PROJECT_ID.gen_ai_batch_prediction.predictions_TIMESTAMP` will be used for BigQuery source.\n",
+        "\n",
+        "This tutorial uses a **BigQuery** table as an example.\n",
+        "\n",
+        "- You can specify the URI of your BigQuery table in `BQ_OUTPUT_URI`, or\n",
```
Comment (holtskinner) [inline gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:390]: Can you add a specification that the notebook has code to create the dataset? When I first read it, I thought you meant that the API service itself creates a dataset if not provided.

**Review by holtskinner — APPROVED** [review]

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb
Comment (gericdong) [reply gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:1]: Discussed with the product team and decided to have two separate notebooks.

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb (hunk 2 above)
Comment (gericdong) [reply gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:328]: Yes

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb (hunk 3 above)
Comment (gericdong) [reply gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb:390]: Updated. Thanks!

File: gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb (hunk 1 above)
Comment (gericdong) [reply gemini/batch-prediction/intro_batch_prediction_using_bigquery_input.ipynb]: Corrected. Thanks!

**Pull request merged**